natss/pkg/apis/messaging/v1alpha1/natss_channel_types.go

```diff
@@ -68,10 +68,10 @@ type NatssChannelStatus struct {
 	// provided targets from inside the cluster.
 	//
 	// It generally has the form {channel}.{namespace}.svc.{cluster domain name}
-	duckv1alpha1.AddressStatus `json:",inline"`
+	duckv1.AddressStatus `json:",inline"`
 
 	// Subscribers is populated with the statuses of each of the Channelable's subscribers.
-	eventingduck.SubscribableTypeStatus `json:",inline"`
```

Comment:
this is breaking change?

Reply:
I think so. I'll open an issue to track it. However there is a risk that no one will have time to work on it before the next release. 

Maybe it's time to get rid of eventing-contrib?